.ci/jenkins/ubuntu-16_04.dockerfile

```diff
@@ -0,0 +1,52 @@
+FROM ubuntu:16.04
+
+ARG HOME=/root
+ARG BUILD_CORES_NUMBER=8
+ARG http_proxy
+ARG https_proxy
+ENV http_proxy ${http_proxy}
+ENV https_proxy ${https_proxy}
+
+# nGraph dependencies
+RUN apt-get -y update && \
+    apt-get -y install git build-essential cmake clang-3.9 git curl zlib1g zlib1g-dev libtinfo-dev && \
+    apt-get clean autoclean && \
+    apt-get autoremove -y
+
+# Python dependencies
+RUN apt-get -y install python python3 \
+                       python-pip python3-pip \
+                       python-dev python3-dev \
+                       python-virtualenv && \
+    apt-get clean autoclean && \
+    apt-get autoremove -y
+
+RUN pip install --upgrade pip setuptools wheel && \
+    pip3 install --upgrade pip setuptools wheel
+
+# ONNX dependencies
+RUN apt-get -y install protobuf-compiler libprotobuf-dev && \
+    apt-get clean autoclean && \
+    apt-get autoremove -y
+
+# Install nGraph
+WORKDIR ${HOME}
+RUN git clone https://github.com/NervanaSystems/ngraph.git && \
+    mkdir ngraph/build && \
+    cd build && \
+    cmake ../ -DNGRAPH_USE_PREBUILT_LLVM=TRUE && \
+    make -j ${BUILD_CORES_NUMBER} && \
```

Comment:
Isn't an easier solution to use `make -j"$(nproc)"` instead of external argument?